spec/support/feature_helpers.rb

```diff
@@ -15,6 +15,7 @@ def stub_search_and_single_result(search, query, result = nil)
     end
   end
 
+  # Deprecated: replace with stub_find_all or stub_find_all_with_pagination
```

Comment:
Maybe add the `# TODO: Y24-190` prefix to this comment so we can later find it when removing v1 code